common/static_modifiers/num_of_provinces.txt

```diff
@@ -5,5 +5,5 @@
 
 num_of_provinces = {
 	monthly_militarized_society = -0.015
-	global_autonomy = 0.001
+	#global_autonomy = 0.001
```

Comment:
Would like to try a reduction instead of removing this modifier altogether.

Reply:
It needs to be prob 100 times weaker. 

There is chance that this can work as well, if it works it will be giving global autonomy per territory.
```
territory_easy_difficulty = { # Penalties for territories on easy difficulty
	yearly_corruption = 0.01
}

territory_normal_difficulty = { # Penalties for territories on normal difficulty
	yearly_corruption = 0.02
}

territory_hard_difficulty = { # Penalties for territories on hard difficulty
	yearly_corruption = 0.02
}

territory_very_hard_difficulty = { # Penalties for territories on very hard difficulty
	yearly_corruption = 0.02
}

territory_ai_difficulty = { #Penalties for territories owned by AI regardless of difficulty.
	yearly_corruption = 0.02
}
```

Reply:
Okay, moving the penalty over to territories makes sense to me.

Something like global_autonomy = 0.01